### ISSUE TITLE
Add a overview for store page notes

### ISSUE THREAD

**Issue by AbteuU** [tfedor/AugmentedSteam#429]
As the title says, i want to suggest this feature so one does have a overview of all the notes made to store pages, so that they are easy to find, not having to browse each individual store page with a note on it.

**Comment by Revadike** [comment]
Maybe in the options page?

**Comment by candela97** [comment]
@MxtOUT Maybe this can be done alongside the notes storage fix?
Could also use this section to display current storage location/quota info.

Another problem is we only store the appid related to the note, which isn't very useful when doing an overview. Maybe store app names in a separate field? Could also let users customize it and default to the one displayed on the store.

**Comment by MxtOUT** [comment]
I wouldn't like to store the name in the database entries.
Is there a cheap way to fetch the app names using the appid?
There exists an easy way for images (https://cdn.cloudflare.steamstatic.com/steam/apps/${appid}/capsule_sm_120.jpg)

**Comment by Revadike** [comment]
> Is there a cheap way to fetch the app names using the appid?

https://api.steampowered.com/ISteamApps/GetAppList/v2/

**Comment by MxtOUT** [comment]
That's cool, but I more thought of only fetching the required mappings, i.e. a selected list of appids

**Comment by Revadike** [comment]
> That's cool, but I more thought of only fetching the required mappings, i.e. a selected list of appids

Sorry, this is the only way, unless you make a backend API.

**Comment by Jleagle** [comment]
https://store.steampowered.com/api/appdetails?appids=440 ?

**Comment by Revadike** [comment]
Of course, the storefront api. I'm stupid. Good call, you can also use it in combination with filters, so it saves data bandwidth:
https://store.steampowered.com/api/appdetails?appids=440&filters=basic

**Comment by MxtOUT** [comment]
That's still a lot of overhead.
A backend solution would be the most appropriate thing here imo

**Comment by gab** [comment]
Since the only way to get synced notes currently is to have a very limited amount of them, an overview is very much needed to do some cleanup. Otherwise you're pretty much backed into a corner.

**Comment by candela97** [comment]
There's currenly no way to export notes stored in IDB, so we should also add the ability to export all notes in JSON format, as suggested in #1676.

Or maybe we can read IDB and include notes in exported settings too? And when importing if the notes exceed quota limits we automatically suggest and save them in IDB.

**Comment by candela97** [comment]
> Steam now lets you put notes on games in your library. I dunno if they exposed an API for that, but if it's a simple dictionary of game ID to text field, perhaps it could be used as a replacement backend for Augmented Steam's notes feature?

Haven't figured out that yet... Debugging the client is a hassle, hope somebody else has. But I think Steam's is more of a client feature and they'll likely keep it that way, since there're more than enough "notes" extensions for browsers out there already (and ours is no different).

**Comment by Bobogoobo** [comment]
Hello, is this still planned? I really need a way to modify or delete notes that are associated with apps that no longer have a store page (banned, etc). Currently the only way would be to export AS settings, edit the file, and import, which I'd rather avoid if possible. And I'm always at 95%+ of storage. Last I checked, changes made on the notes page on ITAD are not synced back, so that's not an option either.
